admin/schema.php

```diff
@@ -884,6 +884,27 @@
 
 # Release marker: 1.3.0
 
+# Count / occurrences / votes 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'bug' ), "
+	votes					I		NOTNULL DEFAULT '1'",
```

Comment:
I wonder if we should start with a vote of 0, so that votes represents someone other than the reporter indicating a +1.  Otherwise, we end up with a case where some votes are captured in history and others are not.